scripts/release-notes.py

```diff
@@ -367,30 +371,30 @@ def spin():
 print("## %s %d, %d" % (today.strftime("%B"), today.day, today.year))
 print()
 
-print("This release includes %d merged PRs by %s author%s." %
-      (len(allprs),
-       len(individual_authors), (len(individual_authors) != 1 and "s" or ""),
-      ))
-
-ext_contributors = individual_authors - crdb_folk
-if len(ext_contributors) > 0:
-    ext_contributors = sorted(ext_contributors)
-    print("We would like to thank the following contributors from the CockroachDB community:")
-    print()
-    for a in ext_contributors:
-        print("-", a)
-    print()
-# Note: CRDB folk can be first-time contributors too, so
-# not part of the if ext_contributors above.
-if len(firsttime_contributors) > 0:
-    print("With special thanks to first-time contributors ", end='')
-    for i, n in enumerate(firsttime_contributors):
-        if i > 0 and i < len(firsttime_contributors)-1:
-            print(', ', end='')
-        elif i > 0:
-            print(' and ', end='')
-        print(n, end='')
-    print('.')
+## Print the release notes sign-up and Downloads section.
+print("""Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-alpha.20180122.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
```

Comment:
1) You can auto-generate the download links by using the `--until` argument perhaps?
2) I would recommend making the download section conditional on a command-line argument too.

Reply:
Done. PTAL.